### PR TITLE
New Azure Databricks All-In-One Template for Vnet Injection with Egress Load Balancer

### PR DESCRIPTION
**Pull request by sunainakrishnamoorthy**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

* Added New All-In-One Template for Vnet Injection with Egress Load Balancer
* Adding README, metadata and parameters files

### CODE REVIEW

File: 101-databricks-all-in-one-template-for-vnet-injection-with-load-balancer/GettingStarted.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+**Location** Location of Data Center
+**Vnet Cidr** Cidr Range of the Vnet
+**Private Subnet Cidr** Cidr Range of the Private Subnet
+**Public Subnet Cidr** Cidr Range of the Public Subnet
+**Load Balancer Backend Pool Name** Name of the Backend Pool of the Load Balancer
+**Load Balancer Frontend Config Name** Name of the Frontend Load Balancer configuration
+**Load Balancer Name** Name of the Load Balancer
+**Load Balancer Public IP Name** Name of the outbount Load Balancer Public IP
+
+## Deployment
```
Comment (bmoore-msft) [inline 101-databricks-all-in-one-template-for-vnet-injection-with-load-balancer/GettingStarted.md:33]: remove deployment section and more the rest to the readme.md

File: 101-databricks-all-in-one-template-for-vnet-injection-with-load-balancer/README.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+### What is Azure Databricks?
+
+Azure Databricks is an Apache Spark-based analytics platform optimized for the Microsoft Azure cloud services platform. Designed with the founders of Apache Spark, Databricks is integrated with Azure to provide one-click setup, streamlined workflows, and an interactive workspace that enables collaboration between data scientists, data engineers, and business analysts.
+
+Azure Databricks is a fast, easy, and collaborative Apache Spark-based analytics service. For a big data pipeline, the data (raw or structured) is ingested into Azure through Azure Data Factory in batches, or streamed near real-time using Kafka, Event Hub, or IoT Hub. This data lands in a data lake for long term persisted storage, in Azure Blob Storage or Azure Data Lake Storage. As part of your analytics workflow, use Azure Databricks to read data from multiple data sources such as Azure Blob Storage, Azure Data Lake Storage, Azure Cosmos DB, or Azure SQL Data Warehouse and turn it into breakthrough insights using Spark.
+
+This template allows you to install the following options
```
Comment (bmoore-msft) [inline 101-databricks-all-in-one-template-for-vnet-injection-with-load-balancer/README.md:25]: this is not proper markdown - use a list to format the list

**Review by bmoore-msft — CHANGES_REQUESTED** [review]
Also see the pipeline results for other changes that need to be made

File: 101-databricks-all-in-one-template-for-vnet-injection-with-load-balancer/GettingStarted.md (hunk 1 above)
Comment (sunainakrishnamoorthy) [reply 101-databricks-all-in-one-template-for-vnet-injection-with-load-balancer/GettingStarted.md]: done

File: 101-databricks-all-in-one-template-for-vnet-injection-with-load-balancer/README.md (hunk 2 above)
Comment (sunainakrishnamoorthy) [reply 101-databricks-all-in-one-template-for-vnet-injection-with-load-balancer/README.md:25]: added list formatting

Comment (sunainakrishnamoorthy) [reply 101-databricks-all-in-one-template-for-vnet-injection-with-load-balancer/README.md:25]: done

**Pull request merged**